Source/Widgets/InfoBox/InfoBox.js

```diff
@@ -200,5 +235,22 @@ click: function () { closeClicked.raiseEvent(this); }');
         return destroyObject(this);
     };
 
+    InfoBox.stringToHtml = function stringToHtml(s, frame) {
```

Comment:
Use documentation to indicate this is private. Sorry, I should have pointed you to the [Documentation Guide](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/DocumentationGuide#private) earlier.

Reply:
Done.